### DISCUSSION TITLE
Quickmove popup empty in TB 128.5.2esr (Linux)

### DISCUSSION THREAD

**Discussion by elhennig** [RealRaven2000/QuickFolders#520] [General]
Since I updated Thunderbird the quickmove popup is empty. How can I debug this to get the root cause for this?

**Comment by RealRaven2000** [comment]
It would be helpful to **switch on "Debug mode" in QuickFolders** and then looking at the JavaScript error console (CTRL+SHIFT+J) when the move / copy gets "stuck". Maybe we will see a warning or exception that adds some technical information as to why this is happening.

In order to set up debug mode (before you test quickMove getting stuck) open QF Settings >> advanced:

![image](https://github.com/user-attachments/assets/68af94e9-8995-4695-9ec2-40b584fb46ee)

enable debug mode and if you want even more info on quickMove specifically, click on the settings wheel and also enable 

```
extensions.quickfolders.debug.quickMove = true
```

![image](https://github.com/user-attachments/assets/c3840001-536c-48f4-ba02-a4aa1b33ded5)

this will add more context in case an error happens during moving / copying messages.

for creating a helpful debug log

1 - reset the contents after Thunderbird has loaded

![image](https://github.com/user-attachments/assets/0ed9b32b-8ba7-4ea9-a4c0-ac8423c5f9f6)

2 - if you can reset the contents also before the error occurs so it doesn't get too long
3 - use the right-click menu to "Copy all messages":

![image](https://github.com/user-attachments/assets/8affb169-b409-42f9-94ac-3b8d270de1fb)

this exports the complete log in text format, which is superior to taking screenshots of errors because it can be text searched and analysed!

**Comment by elhennig** [comment]
The error I get repeatedly is 
`: Component returned failure code: 0x80550013 [nsIMsgLocalMailFolder.createLocalSubfolder]
    verify resource:///modules/SmartMailboxUtils.sys.mjs:144
    getSmartMailbox resource:///modules/SmartMailboxUtils.sys.mjs:267
    getUnifiedFolder chrome://messenger/content/parent/ext-folders.js:1277
    result resource://gre/modules/ExtensionParent.sys.mjs:1221
    withCallContextData resource://gre/modules/ExtensionParent.sys.mjs:664
    result resource://gre/modules/ExtensionParent.sys.mjs:1220
    withPendingBrowser resource://gre/modules/ExtensionParent.sys.mjs:674
    result resource://gre/modules/ExtensionParent.sys.mjs:1219
    callAndLog resource://gre/modules/ExtensionParent.sys.mjs:1170
    recvAPICall resource://gre/modules/ExtensionParent.sys.mjs:1218
SmartMailboxUtils.sys.mjs:162:17
`

**Comment by RealRaven2000** [comment]
can we have full context? what happened before? what happened after? 

Also I do not use the Web APIs for accessing folders, nor do I access a module  SmartMailboxUtils.sys.mjs. 

- what other mail extensions have you installed?
- other than IMAP and POP3 accounts do you have any special mail protocols (such as OWL) ?

**Reply by elhennig** [reply]
I guess I mixed up two addons, sorry for the confusion caused by that

**Reply by RealRaven2000** [reply]
> I guess I mixed up two addons, sorry for the confusion caused by that

can I close this issue as resolved?

**Reply by elhennig** [reply]
Yes, sorry, I missed to tell...
